w3sper.js/tests/harness.js

```diff
@@ -85,11 +99,11 @@ export class Treasury {
     })) {
       for (let i = 0; i < this.#users.length; i++) {
         const userNotes = this.#notes.get(this.#users[i].address.toString());
-        this.#notes.set(
-          this.#users[i].address.toString(),
-          new Map([...userNotes, ...notes[i]]),
-        );
+        mergeMap(userNotes, notes[i], this.#keySet);
+
+        // this.#notes.set(this.#users[i].address.toString(), userNotes);
```

Comment:
Leftover commented code

Reply:
No need to change anything: as a nitpick I would add a comment in the `nonce` situation above, but that's it.

It can be done in the next iteration.